charmtools/build/tactics.py

```diff
@@ -491,6 +501,14 @@ def combine(self, existing):
 
     def apply_edits(self):
         super(MetadataYAML, self).apply_edits()
+        # Remove the merged maintainers from the self.data
+        self.data.pop('maintainer', None)
+        self.data.pop('maintainers', [])
+        # Set the maintainer and maintainers only from this layer.
+        if self.maintainer:
+            self.data['maintainer'] = self.maintainer
+        if self.maintainers:
```

Comment:
Since we're already in here wigglin' about, I'd prefer to see us "deprecate" maintainer and instead put everything as a list under maintainers. There by, if there was a maintainer , it'd become a one item list under maintainers.